lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -1548,14 +1581,17 @@ static uint32_t LookupSymbolInModule(CommandInterpreter &interpreter,
       Symbol *symbol = symtab->SymbolAtIndex(match_indexes[i]);
       if (symbol) {
         if (symbol->ValueIsAddress()) {
+          // Using the new dump function to add colors in the summary.
```

Comment:
If these are for your own info when developing, cool. It's a nice habit to get into, but later we'll go through and remove all these that are "obvious" in production code.

One thing I do is `// TODO: ...`. Then before I push for review I can do `git show | grep TODO` and see if I forgot anything.